### PR TITLE
enhancement(env): add usage example plus add all default vars

### PR DESCRIPTION
**Pull request by kneal**

### CODE REVIEW

File: content/usage/getting-started/environment.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title:  "Using the Environment"
+toc:  true
+weight:  7
+description: >
+  Learn about how to leverage the environment within your builds.
+---
+
+Vela provides the ability to define environment variables scoped to individual steps, services and secrets. Pleas note the environment is design to be unique per container. Vela does inject a variety of [default values]() from build, repo and user information.
```
Comment (JordanSussman) [inline content/usage/getting-started/environment.md]: Looks like you forgot to include the link for `[default values]`.

Comment (kneal) [reply content/usage/getting-started/environment.md]: Yeah had a change of heart on formatting, should be linked below in the list format. I'll remove this markdown formatting

File: content/concepts/pipeline/steps/environment.md [hunk 2]
```diff
@@ -45,69 +45,118 @@ This pipeline will execute the `test` step first, then run the `build` step.
 
 ## Defaults
 
-The following environment variables are injected into every step:
+The following environment variables are injected into every service:
```
Comment (jbrockopp) [inline content/concepts/pipeline/steps/environment.md]: I think we want this to be `step` here 👍 

At least, the filename makes me believe it should be `step` here.

**Review by jbrockopp — CHANGES_REQUESTED** [review]

**Review by jbrockopp — DISMISSED** [review]
LGTM

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by wass3r — APPROVED** [review]

**Pull request merged**